analyzer/src/main/kotlin/PackageJsonUtils.kt

```diff
@@ -79,7 +84,15 @@ internal class PackageJsonUtils {
         }
 
         private fun isYarnWorkspaceRoot(definitionFile: File): Boolean {
-            return definitionFile.readValue<ObjectNode>()["workspaces"] != null
+            return try {
+                definitionFile.readValue<ObjectNode>()["workspaces"] != null
+            } catch(e: JsonProcessingException) {
+                e.showStackTrace()
+
+                log.error { "Could not parse ${definitionFile.invariantSeparatorsPath}: ${e.message}" }
```

Comment:
Nit: Single-quote the path (in case it contains spaces)? Also below...

Reply:
Done.